packages/jest/package.json

```diff
@@ -24,7 +24,7 @@
     "stylis": "4.0.13"
   },
   "peerDependencies": {
-    "@types/jest": "^26.0.14 || ^27.0.0",
+    "@types/jest": ">=26.0.14",
```

Comment:
Supporting `>=` is a claim that we can't really guarantee as the future is unknown 😅 

```suggestion
    "@types/jest": "^26.0.14 || ^27.0.0 || ^28.0.0",
```

Reply:
Btw. this is somewhat interesting as Jest 28 comes with builtin types as it has been rewritten in TS:
https://github.com/facebook/jest/blob/3a8696bcf0e02a5f59080123d1a20e8b6d66d8df/packages/jest/package.json#L6
but I also see that `@types/jest` exists for v28:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dfbb6552078413b8638e55160227ca2f2e298259/types/jest/index.d.ts#L1

The relation between both is somewhat unclear to me. @SimenB would you mind commenting on this one? Is there any specific purpose for which `@types/jest` continue to exist today?

Reply:
Long term it should just provide globals, but blocked by time and effort. Latest remains https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44365